src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -249,6 +258,22 @@ private boolean isIntegralPrimitiveType(Object incrementingColumnValue) {
            || incrementingColumnValue instanceof Byte;
   }
 
+  /**
+   * @param columnName column name
+   * @param columnAlias optional column alias
+   * @param quoteString string used to quote SQL identifiers
+   * @return the quoted column with a prefixed alias, if applicable
+   */
+  private String getQuotedColumn(String columnName, String columnAlias, String quoteString) {
+    String result = "";
+
+    if (!columnAlias.isEmpty()) {
+      result += JdbcUtils.quoteString(columnAlias, quoteString) + ".";
```

Comment:
Yeah, exactly. Just seems simpler and more intuitive for the user. My only concern is if we then also need to deal with the user doing their own quoting since it looks like, e.g., `.` is valid for table names as long as it is quoted.